cl_sii/rcv/data_models.py

```diff
@@ -214,79 +227,109 @@ def as_dte_data_l2(self) -> cl_sii.dte.data_models.DteDataL2:
         return dte_data
 
 
-@dataclasses.dataclass(frozen=True)
+@pydantic.dataclasses.dataclass(
+    frozen=True,
+    config=type('Config', (), dict(
+        arbitrary_types_allowed=True,
+    ))
+)
 class RvDetalleEntry(RcvDetalleEntry):
 
     """
     Entry of the "detalle" of an RV ("Registro de Ventas").
     """
 
+    ###########################################################################
+    # constants
+    ###########################################################################
+
+    DATETIME_FIELDS_TZ = SII_OFFICIAL_TZ
+
     RCV_KIND = RcvKind.VENTAS
     RC_ESTADO_CONTABLE = None
 
     # TODO: docstring
     # TODO: can it be None? What happens for those "tipo docto" that do not have a receptor?
-    receptor_razon_social: str = dc_field()
+    receptor_razon_social: str
 
     # TODO: docstring
     # note: must be timezone-aware.
-    fecha_acuse_dt: Optional[datetime] = dc_field()
+    fecha_acuse_dt: Optional[datetime] = None
```

Comment:
The field is `Optional`, but it doesn't have a default value (`dc_field()` is not a "real" default value). The effect of removing the default value (i.e. removing ` = None`) is that a value must always be provided for the field, but that value is allowed to be `None` (in other words, _"explicit `None`"_ instead of _"default `None`"_).

```suggestion
    fecha_acuse_dt: Optional[datetime]
```

Reply:
@jtrobles-cdd thanks for the explanation! I really have no idea why I made this change and added the default value `None`, but good thing you noticed.